.github/workflows/rust.yml

```diff
@@ -25,21 +35,44 @@ jobs:
           ~/.cargo/registry/index/
           ~/.cargo/registry/cache/
           ~/.cargo/git/db/
-          target/            
-        key: ${{ runner.os }}-debug-${{ hashFiles('**/Cargo.lock') }}
-        restore-keys: ${{ runner.os }}-debug-
-    - run: cargo fmt --check
+          target/
+        key: ${{ runner.os }}-check-${{ hashFiles('**/Cargo.lock') }}
+        restore-keys: ${{ runner.os }}-check-
+    - run: cargo check --verbose
```

Comment:
Doesn't cargo test already perform a cargo check ?

Don't you want to check formatting with fmt ?

Reply:
It does, you're right

Reply:
I just remove this redundant step on master, thx for pointing it out